cli/spec/kontena/cli/grid_options_spec.rb

```diff
@@ -0,0 +1,58 @@
+require "kontena/cli/common"
+require "kontena/cli/grid_options"
+require 'json'
+
+describe Kontena::Cli::GridOptions do
+  let(:subject) do
+    Class.new(Kontena::Command) do
+      include Kontena::Cli::Common
+      include Kontena::Cli::GridOptions
+      def execute
+        puts "grid:#{current_grid}"
+        puts "master:#{current_master.name}"
+      end
+    end.new('')
+  end
+
+  before(:each) do
+    RSpec::Mocks.space.proxy_for(File).reset
+    allow(File).to receive(:readable?).and_return(true)
+    allow(File).to receive(:exist?).and_return(true)
+    allow(File).to receive(:read).and_return(
+      {
+        'current_server' => 'alias',
+        'servers' => [
+           {'name' => 'some_master', 'url' => 'some_master', 'grid' => 'somegrid'},
+           {'name' => 'alias', 'url' => 'someurl', 'grid' => 'somegrid'}
+        ]
+      }.to_json
+    )
+    Kontena::Cli::Config.reset_instance
+  end
+
+
+  describe '--grid' do
+    it 'sets the current_grid' do
+      expect{subject.run(['--grid', 'foogrid'])}.to output(/grid:foogrid/).to_stdout
+      expect{subject.run(['--grid', 'foogrid'])}.to output(/master:alias/).to_stdout
```

Comment:
All of these specs are actually broken and fail with `SystemExit`, but somehow rspec is still passing?

```
DEBUG Running  with ["--grid", "foogrid"] -- callback matcher = 'nil'
DEBUG Loading configuration from /tmp/.kontena_client.json
DEBUG Configuration loaded with 2 servers.
DEBUG Current master: alias
DEBUG Current grid: somegrid
 [error] You are not logged into a Kontena Master. Use: kontena master login
Got SystemExit: exit - Exit code: 1
.
```